samples/contrib/versioned-pipeline-ci-samples/mnist-ci-sample/tensorboard/component.yaml

```diff
@@ -0,0 +1,14 @@
+name: tensorboard visualization
+description: visualize training in tensorboard
+inputs:
+  - {name: logdir, type: string}
+outputs:
+  - {name: MLPipeline UI metadata, type: UI metadata}
+implementation:
+  container:
+    image: tensorboard_image_location
+    command: ['python', '/tensorboard.py']
+    args: ['--logdir', {inputValue: logdir}]
+    fileOutputs: 
+      MLPipeline UI metadata: /mlpipeline-ui-metadata.json
```

Comment:
It's better to have all paths configurable.

Reply:
OK